preview/MsixCore/msixmgr/ByteBuffer.hpp

```diff
@@ -0,0 +1,157 @@
+
+#pragma once
+
+#include <assert.h>
+
+#define APPXCOMMON_NEW_TAG (std::nothrow)
+#define DebugFillMemory(pv, cb)
+
+namespace MsixCoreLib
+{
+    struct COMMON_BYTES
+    {
+        ULONG length;
+        __field_ecount(length) BYTE* bytes;
+    };
+
+    class ByteBuffer

```

Comment:
>ByteBuffer [](start = 10, length = 10)

where's this used? do you need it?

Reply:
This is used in CryptoProvider.hpp at https://github.com/microsoft/msix-packaging/blob/0c846130c19da326f5b41c93738506cf55569441/preview/MsixCore/msixmgr/CryptoProvider.hpp#L18 to create hashObjectBuffer and digestBuffer.

Reply:
can you use std:: types instead?

---
In reply to: [311285939](https://github.com/microsoft/msix-packaging/pull/186#discussion_r311285939) [](ancestors = 311285939)

Reply:
Deleted ByteBuffer and replaced its usage with std::vector<BYTE>.